src/app/shared/figure/figure.component.scss

```diff
@@ -0,0 +1,25 @@
+@import "const/_variables.scss";
```

Comment:
Use sass correctly 
In import you should not use _ in the beginning and file extension
`@import "const/variables";`

Reply:
fix